Sources/AppcuesKit/Presentation/Traits/Appcues/AppcuesStepTransitionAnimationTrait.swift

```diff
@@ -9,6 +9,10 @@
 import UIKit
 
 internal class AppcuesStepTransitionAnimationTrait: ContainerDecoratingTrait {
+    struct Config: Decodable {
+        let duration: Double?
+        let easing: String?
```

Comment:
I think the type here should be able to be `Easing?`, since that's also `Decodable`.